src/test/resources/hw1/AllTestsSuite.xml

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+<suite name="All tests suite">
+
+    <test name="Additive and multiplicative tests">
+        <classes>
+            <class name="hw1.CalculatorSumTest"></class>
+            <class name="hw1.CalculatorSubtractTest"></class>
+            <class name="hw1.CalculatorMultiplyTest"></class>
+            <class name="hw1.CalculatorDivideTest"></class>
+        </classes>
+    </test>
+</suite>
```

Comment:
Why do you decide include set of all classes to common suite? Do we have other way add all tests to the main suite?

Reply:
Something like that? 

<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
<suite name="All tests suite">

    <test name="Sum test">
        <classes>
            <class name="hw1.CalculatorSumTest"></class>
        </classes>
    </test>

    <test name="Subtract test">
        <classes>
            <class name="hw1.CalculatorSubtractTest"></class>
        </classes>
    </test>

    <test name="Multiply test">
        <classes>
            <class name="hw1.CalculatorMultiplyTest"></class>
        </classes>
    </test>

    <test name="Divide test">
        <classes>
            <class name="hw1.CalculatorDivideTest"></class>
        </classes>
    </test>
</suite>

Reply:
What about this suite, it's because of the following test requirement is "3.	Create test suite, which could run all tests"

Reply:
Or maybe this way to combine all tests? But there is no difference between these approaches.. 

<suite-files>
        <suite-file path="Group1TestSuite.xml"></suite-file>
        <suite-file path="Group2TestSuite.xml"></suite-file>
    </suite-files>

Reply:
Or maybe this way to combine the tests? But there is no difference between these approaches..
```
<suite name="All tests suite">

   <suite-files>
        <suite-file path="Group1TestSuite.xml"></suite-file>
        <suite-file path="Group2TestSuite.xml"></suite-file>
    </suite-files>

</suite> 
```